src/components/List/List.css

```diff
@@ -49,7 +49,6 @@
   align-items: center;
   background: var(--list-background-color);
   border: var(--list-border);
-  border-radius: var(--list-border-radius);
```

Comment:
Since we are not using these variables anymore, shouldn't we remove these from `:root`?

Reply:
hey @Rheniery my approach here was wrong. I'll be sending the correct one shortly.